AY2122S2-CS2103T-T13-3/tp
# Implement command for adding meeting entry

## Related issues

Closes #33 

## Description


Added AddCommandParser to parse input which creates an AddCommand object to be executed.
Added the add command option into the switch case in LinkyTimeParser.

## How to test

`add n/Lecture u/https://zoom.com d/today m/CS2103 r/Y`
Make sure the URL is fully formed, containing the scheme as well, e.g. https://

## Other information

Need to remove test cases from AddCommandIntegrationTest once ready to make full transition to LinkyTime.
Implement execute_duplicateMeetingEntry_throwsCommandException() once we are able to retrieve meetinglist.
Implement parse_invalidValue_failure() in AddCommandParserTest once we do further input validation.
